<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuraciones</class>
 <widget class="QDialog" name="Configuraciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuraciones</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes/iconos.qrc">
    <normaloff>:/opciones.png</normaloff>:/opciones.png</iconset>
  </property>
  <widget class="QListWidget" name="listaOpciones">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>371</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Generales</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Editor de Texto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funciones Auxiliares</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="panelPrincipal">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>511</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Legus IDE</string>
     </property>
     <widget class="QCheckBox" name="chkVentanaDeBienvenida">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar Ventana de Bievenida</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Vista en Pestañas</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>481</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Fuentes</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Fuente:</string>
      </property>
     </widget>
     <widget class="QFontComboBox" name="cmbFuente">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>193</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Tamaño:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbTamanio">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>36</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>48</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>72</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>481</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Colores</string>
     </property>
     <widget class="QListWidget" name="listaColoreado">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>341</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListView::item:hover {
	background-color: rgb(140, 140, 255);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnForeground">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBackground">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3"/>
  </widget>
  <widget class="QPushButton" name="btnAplicar">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>402</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aplicar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>402</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/iconos.qrc"/>
 </resources>
 <connections/>
</ui>
